.util/pylint_scores.txt

```diff
@@ -1,3 +1,19 @@
+************* Module login
+Dashboard/login.py:12:0: C0301: Line too long (459/100) (line-too-long)
+
+-----------------------------------
+Your code has been rated at 9.83/10
+
+************* Module 1_price
+Dashboard/pages/1_price.py:1:0: C0103: Module name "1_price" doesn't conform to snake_case naming style (invalid-name)
+Dashboard/pages/1_price.py:35:0: C0116: Missing function or method docstring (missing-function-docstring)
+Dashboard/pages/1_price.py:35:21: W0613: Unused argument 'product_url' (unused-argument)
+Dashboard/pages/1_price.py:39:0: C0116: Missing function or method docstring (missing-function-docstring)
+Dashboard/pages/1_price.py:39:14: W0613: Unused argument 'product_url' (unused-argument)
```

Comment:
Please fix this!!

Reply:
file has to be called 1_price but other things are done